auction-app-frontend/src/components/ProductCard/ProductCard.jsx

```diff
@@ -1,18 +1,22 @@
+import { Link } from "react-router-dom";
+
+import { ROUTE_PATHS } from "src/constants";
+
 import "./style.scss";
 
-const ProductCard = ({ imageUrl, name, startPrice }) => {
+const ProductCard = ({ id, productImages, name, startPrice }) => {
   return (
-    <div className="product-card">
+    <Link to={`${ROUTE_PATHS.PRODUCT_DETAILS}/${id}`} className="product-card">
```

Comment:
Just click on image or product name should redirect to the product details page, not whole component